ntoskrnl/include/internal/i386/asmmacro.S

```diff
@@ -249,6 +249,22 @@ MACRO(TRAP_ENTRY, Trap, Flags)
     .ENDP
 ENDM
 
+MACRO(TASK_ENTRY, Trap, Flags)
+    // EXTERN @&Trap&Handler@0 :PROC
+    EXTERN _&Trap&Handler :PROC
+    PUBLIC _&Trap
+    .PROC _&Trap
+        /* Generate proper debugging symbols */
+        FPO 0, 0, 0, 0, 0, FRAME_TSS
```

Comment:
I don't know what kind of "documentation" I can point to, except that for having windbg to correctly follow the stacktrace after a TSS switch has been done, as it's the case when Trap02 or Trap08 are run, you need to use FRAME_TSS instead of FRAME_TRAP (these defines already exist in our code, and are mentioned e.g. [there](https://docs.microsoft.com/en-us/windows/desktop/api/winnt/ns-winnt-_fpo_data)). And, if you don't use FRAME_TSS but use FRAME_TRAP instead, you will see that you cannot reproduce stacktraces with the format similar to the ones shown in the articles I linked to in the description of this PR!